tests/batch_tests/multi_objective_tests/test_study.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+import uuid
+
+import numpy as np
+import pytest
+
+import optuna
+from optuna.batch.multi_objective.trial import BatchMultiObjectiveTrial
+from optuna.study import StudyDirection
+from optuna.testing.storage import StorageSupplier
+
+
+def test_create_study() -> None:
+    study = optuna.batch.multi_objective.create_study(["maximize"])
+    assert study.n_objectives == 1
+    assert study.directions == [StudyDirection.MAXIMIZE]
+
+    study = optuna.batch.multi_objective.create_study(["maximize", "minimize"])
+    assert study.n_objectives == 2
+    assert study.directions == [StudyDirection.MAXIMIZE, StudyDirection.MINIMIZE]
+
+    with pytest.raises(ValueError):
+        # Empty `directions` is not allowed.
+        study = optuna.batch.multi_objective.create_study([])
+
+
+def test_load_study() -> None:
+    with StorageSupplier("sqlite") as storage:
+        study_name = str(uuid.uuid4)
+
+        with pytest.raises(KeyError):
+            # Test loading an unexisting study.
+            optuna.batch.multi_objective.study.load_study(study_name=study_name, storage=storage)
+
+        created_study = optuna.batch.multi_objective.study.create_study(
+            ["minimize"], study_name=study_name, storage=storage
+        )
+
+        # Test loading an existing study.
+        loaded_study = optuna.multi_objective.study.load_study(
+            study_name=study_name, storage=storage
+        )
+        assert created_study._study_id == loaded_study._study_id
+
+
+@pytest.mark.parametrize("n_objectives", [1, 2, 3])
+def test_optimize(n_objectives: int) -> None:
+    directions = ["minimize" for _ in range(n_objectives)]
+    study = optuna.batch.multi_objective.create_study(directions)
+
+    def objective(trial: BatchMultiObjectiveTrial) -> List[np.ndarray]:
+        return [trial.suggest_float("v{}".format(i), 0, 5) for i in range(n_objectives)]
```

Comment:
I think we should note somewhere that the shape of the returned value of the `objective` for the batch multi-objective optimization is `(n_objectives, batch_size)`, and the type of the returned value is not 2-dimensional `np.ndarray` but `List` of 1-dimensional `np.ndarray`. It is kind to users.